README.md

```diff
@@ -744,12 +754,10 @@ When making code changes that may affect performance, please run benchmarks agai
 
 The core library is transpiled using [emscripten](http://kripken.github.io/emscripten-site). The easiest way to build from source locally is by using Docker. Make sure Docker is installed, then:
 
-    yarn init-docker
-    yarn run build-emscripten
-
-The build script uses the `H3_VERSION` file to determine the version of the core library to build.
+    yarn docker-boot
+    yarn build-emscripten
 
-NOTE: The current `h3-js` is built with `emscripten-1.37.40`. Earlier or later versions MAY NOT WORK (emscripten does not follow semver, so patch updates may include breaking changes).
```

Comment:
Are you sure you want to remove this warning about emscripten?

Reply:
It's no longer valid (currently using `v1.38.29`) and the recommended option is now clearly defined in the scripts, when you run `docker boot`.